protocol-contracts/governance/hardhat.config.ts

```diff
@@ -54,14 +54,25 @@ const config: HardhatUserConfig = {
         ],
     },
     networks: {
-        'arbitrum-sepolia': {
-            eid: EndpointId.ARBSEP_V2_TESTNET,
-            url: process.env.RPC_URL_ARB_SEPOLIA || 'https://arbitrum-sepolia.gateway.tenderly.co',
+        'ethereum-mainnet': {
+            eid: EndpointId.ETHEREUM_V2_MAINNET,
+            url: process.env.MAINNET_RPC_URL || '',
             accounts,
         },
-        'base-sepolia': {
-            eid: EndpointId.BASESEP_V2_TESTNET,
-            url: process.env.RPC_URL_BASE_SEPOLIA || 'https://base-sepolia.gateway.tenderly.co',
+        'ethereum-testnet': {
+            eid: EndpointId.SEPOLIA_V2_TESTNET,
+            url: process.env.SEPOLIA_RPC_URL!,
+            accounts,
+        },
+        // No ZAMA_V2_MAINNET endpoint available yet.
+        // 'gateway-mainnet': {
+        //     eid: EndpointId.ZAMA_V2_MAINNET,
+        //     url: process.env.RPC_URL_ZAMA_GATEWAY_MAINNET || '',
+        //     accounts,
+        // },
```

Comment:
fyi you could do it like this

```suggestion
        'gateway-mainnet': {
            // @ts-ignore: TODO: Remove TS ignore once LayerZero endpoint is deployed.
            eid: EndpointId.ZAMA_V2_MAINNET,
            url: process.env.RPC_URL_ZAMA_GATEWAY_MAINNET || '',
            accounts,
        },
```